src/averageAges.js

```diff
@@ -38,6 +56,19 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const women = people
+    .filter(
+      (person) =>
+        person.sex === 'f'
+        && (withChildren
+          ? people.find((child) => child.mother === person.name)
+          : !withChildren)
+    )
+    .map((woman) => woman.died - woman.born);
+
+  const result = callBack(women, women.length);
```

Comment:
Result is not the best name for variable. Try to specify what you get from this function call.
Fix all cases